Segment-Wootric/WTRWootricIntegration.m

```diff
@@ -33,6 +33,18 @@ - (void)identify:(SEGIdentifyPayload *)payload {
   [Wootric setEndUserProperties:endUserProperties];
 }
 
+- (void)track:(SEGTrackPayload *)payload {
+  NSDictionary *properties = payload.properties;
+  NSString *eventName = payload.event;
+  NSString *language = [properties objectForKey:@"language"];
+
+  if (language) {
+    [Wootric setCustomLanguage:language];
+  }
+
+  [Wootric setEventName:eventName];
```

Comment:
Should there be a check for `eventName` before setting?

Reply:
Not really necessary. The [track](https://segment.com/docs/connections/sources/catalog/libraries/mobile/ios/#track) call requires the eventName to be not-null.

Reply:
Cool.